pymess/backend/dialer/daktela.py

```diff
@@ -0,0 +1,100 @@
+from chamber.utils.datastructures import ChoicesEnum
+from django.utils import timezone
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext_lazy as _l
+
+from pymess.backend.dialer import DialerBackend
+from pymess.config import settings
+from pymess.models import DialerMessage
+from pymess.utils.logged_requests import generate_session
+
+
+class DaktelaDialerBackend(DialerBackend):
+    """
+    Dialer backend implementing Daktela service https://www.daktela.com/api/v6/models/campaignsrecords
+    """
+
+    STATE = ChoicesEnum(
+        ('NOT_ASSIGNED', _l('not assigned'), 'not assigned'),
+        ('READY', _l('ready'), 'ready'),
+        ('RESCHEDULED_BY_DIALER', _l('rescheduled by dialer'), 'rescheduled by dialer'),
+        ('CALL_IN_PROGRESS', _l('call in progress'), 'call in progress'),
+        ('HANGUP', _l('hangup'), 'hangup'),
+        ('DONE', _l('done'), 'done'),
+        ('RESCHEDULED', _l('rescheduled'), 'rescheduled'),
+    )
+
+    # TODO: clean-up mappings
+    STATES_MAPPING = {
+        STATE.NOT_ASSIGNED: DialerMessage.STATE.UNKNOWN,
+        STATE.READY: DialerMessage.STATE.READY,
+        STATE.RESCHEDULED_BY_DIALER: DialerMessage.STATE.RESCHEDULED,
+        STATE.CALL_IN_PROGRESS: DialerMessage.STATE.IN_PROGRESS,
+        STATE.HANGUP: DialerMessage.STATE.DONE,
```

Comment:
stavy daktely vlastně ukládáme také do těch extra fields, takže tohle zjistíme že?

Reply:
pridam to tam

Reply:
pretoze `DialerMessage` je univerzal, `Daktela` je uz konkretna implementacia s vlastnymi stavmi